guides/common/modules/con_converting-a-host-to-rhel.adoc

```diff
@@ -31,14 +31,15 @@ The duration of the conversion process depends on the number of packages that ha
 
 .Prerequisites
 * Review https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/converting_from_an_rpm-based_linux_distribution_to_rhel/index#con_supported-conversion-paths_converting-from-a-linux-distribution-to-rhel[Supported conversion paths] in _Converting from an RPM-based Linux distribution to RHEL_.
-* You must have completed the steps 1.{range}5. of the procedure https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/converting_from_an_rpm-based_linux_distribution_to_rhel/index#proc_preparing-for-a-rhel-conversion_converting-from-a-linux-distribution-to-rhel[Preparing for a RHEL conversion] in _Converting from an RPM-based Linux distribution to RHEL_.
+* You must have completed the steps 1.{range}5. of the procedure https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/converting_from_an_rpm-based_linux_distribution_to_rhel/index#proc_preparing-for-a-rhel-conversion_converting-using-the-command-line[Preparing for a RHEL conversion] in _Converting from an RPM-based Linux distribution to RHEL_.
 * Ensure you have a subscription manifest uploaded to your {Project} and that there are sufficient {RHEL} entitlements allocated for the conversions you intend.
 Alternatively, you can use Ansible variables to tell the role to import the manifest from disk.
 The manifest must be imported to the organization to which you will register hosts for conversion.
 +
 You can update your allocations and download the updated manifest from the https://access.redhat.com[Red Hat Customer Portal].
 For more information, see https://access.redhat.com/documentation/en-us/red_hat_subscription_management/2022/html/using_red_hat_subscription_management/using_manifests_con[Using manifests] in _Red Hat Subscription Management_.
-* Ensure that you have enabled Red Hat repositories in {Project} for the minor {RHEL} version to which you convert your hosts.
+* Ensure that you have enabled and synchronized Red Hat repositories in {Project} for the minor {RHEL} version to which you convert your hosts.
+For more information, see {ContentManagementDocURL}Enabling_Red_Hat_Repositories_content-management[Enabling Red Hat Repositories] and {ContentManagementDocURL}Synchronizing_Repositories_content-management[Synchronizing Repositories] in _{ContentManagementDocTitle}_.
```

Comment:
Will two links in one line work? I remember there was an issue about this in the past.

Reply:
The problem was with xrefs. Normal links should be fine. (I hope)